content/community/roadmap.md

```diff
@@ -1,21 +0,0 @@
----
-title: "Roadmap"
-date: 2018-07-16T14:49:14-07:00
-draft: false
-weight: 100
-aliases: [/roadmap]
----
-Read OpenCensus’s journey ahead: [platforms and languages](https://opensource.googleblog.com/2018/05/opencensus-journey-ahead-part-1.html).
-
-
-#### Languages
-
-{{<languages>}}
```

Comment:
We definitely need this, why was it removed?

Reply:
I want to move and merge this section into the languages page

Reply:
Re-added the file